backend/api/views/simplePostView.py

```diff
@@ -38,14 +38,15 @@ def createNewPost(request, author_id):
 # pageNum, optional, int
 #####################################
 def getPostsByAuthorId(request, author_id, pageNum = 0):
-  request.data['author_id'] = author_id
+  if 'pageNum' in request.data.keys():
+    pageNum = int(request.data.dict()['pageNum'])
   res = postServices.getPostByAuthorId(request, author_id)
   if res.status_code == 404:
     return res
   pagedRes = postServices.getPaginatedPosts(res, pageNum).data
```

Comment:
i think this is fine for now but we should move to using the pagination class provided by django, it error handles for us

Reply:
My struggle is mostly from how to get the page number. It is not known yet whether it comes from the body or param.